core/src/main/resources/AppConstants.properties

```diff
@@ -694,3 +742,63 @@ message.max.memory.size=5242880
 migration.rewriteLegacyClassNames=true
 
 ####
+####
+
+#rest.cors.allowOrigin
+#rest.cors.exposeHeaders
+
+####
+#### Mail
+
+#mail.mime.charset
+#mail.smtp.from
+#mail.smtp.host
+#mail.smtp.starttls.enable
+
+####
+#### MongoDB
+
+#mongodb.datasource.default
+
+####
+#### Cmis
+
+#cmis.datetime.formatstring
+#cmis.mtomfilter.active
+#cmis.mtomproxy.active
+#cmis.mtomproxy.servlet
+#cmis.securityHandler.type
+
+## [Deprecated]
+#cmissender.processproperties.legacydateformat
+
+#cmisbridge.active=true
+#cmisbridge.closeConnection
+#cmisbridge.url
+#cmisbridge.adapterDispatcher
+
+####
+#### Ldap
+
+## [Deprecated]
```

Comment:
Willen we deprecated properties hier wel opnemen?

Reply:
Tja, het gaat er meer om dat de json die hier uit komt ook deze medeleding krijgt dan dat wij er profijt aan hebben.